northbound/rest/src/test/java/org/eclipse/sensinact/northbound/rest/integration/notification/ResourceNotificationsTest.java

```diff
@@ -165,9 +166,11 @@ void resourceNotificationNonExistent() throws Exception {
             // First will be admin friendlyName
             ResourceDataNotification friendlyName = queue.poll(1, TimeUnit.SECONDS);
             assertNotNull(friendlyName);
-            assertEquals("friendlyName", friendlyName.resource,
+            assertEquals(ProviderPackage.Literals.ADMIN__FRIENDLY_NAME.getName(), friendlyName.resource,
```

Comment:
We want this integration test to fail if someone changes the name from `friendlyName` to something else.

Reply:
In that case the literal will be generated differently and you will get a compile error right away